wlutil/wlutil.py

```diff
@@ -335,6 +335,10 @@ def deriveOpts(self):
             m = self.importDistro(dPath)
             self['distro-mods'][m.__name__] = m
 
+        if self['jlevel'] is None:
+            self['jlevel'] = psutil.cpu_count()
```

Comment:
I think there is a case where `psutil.cpu_count()` can return `None` and you should probably handle that by throwing an exception and telling the user to set it in their marshal.yaml.  I still would never use unbounded `-j` because that causes responsiveness problems that you're trying to address here.

Reply:
fixed, defaults to 1 (and a warning) if cpu_count() fails.